public/stylesheets/main.css

```diff
@@ -0,0 +1,141 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Texturina:wght@100&display=swap');
+
+body {
+  font-family: 'Texturina', serif;
+}
+
+/* .container1, .container2, .container3 {
+  width: fit-content;
+  margin: auto;
+} */
+
+.big-container {
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+}
+
+.container1 {
+  margin: 10px;
+  display: flex;
+  background-color: gray;
+  width: 1000px;
+  height: 350px;
+}
+
+.adolescent {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  margin: 20px;
+  background-color: #EDE8E7;
+  flex-basis: 100%;
+}
+
+.theAdolescent1 {
+  margin: 20px;
+  flex-basis: 40%;
+}
+
+.theAdolescent2 {
+  margin: 20px;
+  background-color: #669900;
+  flex-basis: 50%;
+  width: 95%;
+}
+
+.theAdolescent3 {
+  margin: 20px;
+  flex-basis: 40%;
+  width: 40%;
+}
+
+
+.container2 {
+  margin: 10px;
+  display: flex;
+  background-color: gray;
+  width: 1000px;
+  height: 450px;
+}
+
+.child1 {
+  background-image: url('/images/complete.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 20px;
+  background-color: #EDE8E7;
+  flex-basis: 33%;
+}
+.child2 {
+  background-image: url('/images/leper 1.png');
+  background-repeat: no-repeat;
+  background-size:contain;
+  margin: 20px;
+  flex-basis: 33%;
+  background-color: #EDE8E7;
+}
+.child3 {
+  background-image: url('/images/leper complete.png');
+  background-repeat: no-repeat;
+  background-size:contain;
+  margin: 20px;
+  flex-basis: 33%;
+  background-color: #EDE8E7;
+}
+
+.container3 {
+  margin: 10px;
+  display: flex;
+  background-color: gray;
+  width: 1000px;
+  height: 350px;
+}
+
+.juvenile {
+  margin: 20px;
+  background-color: #EDE8E7;
+  flex-basis: 100%;
+}
+
+
```

Comment:
Please remove these extra two new lines here.

Reply:
two lines removed for clarity, working on other requests